languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -22094,113 +22094,6 @@ USA
-                <example correction="either">She isn't coming later <marker>too</marker>.</example>
-                <example>Louise Please say not soon too.</example>
-                <example>I've got the first round to include but not limited too...</example>
-                <example>I don't see any reason why I can't go, too.</example>
-                <example>I also got criticism for living with my mom and not paying rent even though i couldn't afford too.</example>
-                <example>I am not able too.</example>
-                <example>You don't have too.</example>
-                <example>He doesn't have too.</example>
-                <example>I hate not wanting too.</example>
-                <example>I'm not supposed too.</example>
-                <example>We didn't need too.</example>
-                <example>"You're not one, too."</example>
-            </rule>
-            <rule>
-                <pattern>
-                    <token inflected="yes">do</token>
-                    <token inflected="yes">not</token>
-                    <token regexp="yes">have|&verbs_followed_by_infin;</token>
-                    <marker>
-                        <token>too</token>
-                    </marker>
-                    <token postag="SENT_END" />
-                </pattern>
-                <message>Did you mean <suggestion>to</suggestion>?</message>
-                <example correction="to">You don't have <marker>too</marker>.</example>
-                <example correction="to">He doesn't have <marker>too</marker>.</example>
-            </rule>
-            <rule>
-                <pattern>
-                    <token inflected="yes">be</token>
-                    <token inflected="yes">not</token>
-                    <token>able</token>
-                    <marker>
-                        <token>too</token>
-                    </marker>
-                </pattern>
-                <message>Did you mean <suggestion>to</suggestion>?</message>
-                <example correction="to">I am not able <marker>too</marker>.</example>
-            </rule>
-            <rule>
-                <pattern>
-                    <token inflected="yes">not</token>
-                    <token regexp="yes" inflected="yes" postag="VBG">have|&verbs_followed_by_infin;</token>
-                    <marker>
-                        <token>too</token>
-                    </marker>
-                    <token>
-                        <exception postag="JJ" />
-                        <exception>much</exception>
-                    </token>
-                </pattern>
-                <message>Did you mean <suggestion>to</suggestion>?</message>
-                <example correction="to">I hate not having <marker>too</marker>.</example>
-                <example correction="to">I hate not wanting <marker>too</marker>.</example>
-                <example>He isn't trying too hard.</example>
-            </rule>
-        </rulegroup>
```

Comment:
I don't really see a problem with this rule. Maybe it's rather a style rule than a grammar rule?
![Bildschirmfoto 2021-07-19 um 18 43 47](https://user-images.githubusercontent.com/37363/126196528-e41a4580-7966-420a-b12e-8384009a0e00.png)

Reply:
Well then style rule it shall be :)